app/models/user.rb

```diff
@@ -41,7 +41,10 @@ class User < ApplicationRecord
   include User::Omniauthable
   include User::Onboarding
 
+  attr_accessor :feedback
```

Comment:
```suggestion
  attr_accessor :delete_account_feedback
```

Reply:
why do we need this here? is this if there are errors in the password and we render the same text that the user typed out?

Reply:
> why do we need this here? is this if there are errors in the password and we render the same text that the user typed out?

i forgot i could use **fields_for** and nested attributes. have replaced it now.